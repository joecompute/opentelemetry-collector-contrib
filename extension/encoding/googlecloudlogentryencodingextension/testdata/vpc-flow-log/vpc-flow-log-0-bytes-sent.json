{
  "insertId": "16gybk0f1tkpe0",
  "jsonPayload": {
    "packets_sent": "640",
    "bytes_sent": "0",
    "reporter": "SRC",
    "network_service": {
      "dscp": 32
    },
    "connection": {
      "protocol": 6,
      "dest_port": 40708,
      "src_port": 443,
      "dest_ip": "35.191.240.41",
      "src_ip": "10.128.0.41"
    },
    "start_time": "2025-09-27T21:12:02.937646004Z",
    "src_instance": {
      "region": "us-central1",
      "project_id": "elastic-obs-integrations-dev",
      "vm_name": "proxy-crest-new-group-b51k",
      "zone": "us-central1-a",
      "managed_instance_group": {
        "name": "proxy-crest-new-group",
        "zone": "us-central1-a"
      }
    },
    "src_vpc": {
      "vpc_name": "default",
      "project_id": "elastic-obs-integrations-dev",
      "subnetwork_region": "us-central1",
      "subnetwork_name": "default"
    },
    "end_time": "2025-09-27T21:12:02.937646004Z"
  },
  "resource": {
    "type": "vpc_flow_logs_config",
    "labels": {
      "name": "joe-test-flow-logs-default",
      "resource_container": "projects/774712120909",
      "location": "global"
    }
  },
  "timestamp": "2025-09-27T21:12:11.539677207Z",
  "labels": {
    "target_resource_name": "projects/774712120909/global/networks/default"
  },
  "logName": "projects/elastic-obs-integrations-dev/logs/networkmanagement.googleapis.com%2Fvpc_flows",
  "receiveTimestamp": "2025-09-27T21:12:11.539677207Z"
}