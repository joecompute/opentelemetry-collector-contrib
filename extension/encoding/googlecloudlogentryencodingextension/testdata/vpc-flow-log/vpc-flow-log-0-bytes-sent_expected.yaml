resourceLogs:
  - resource:
      attributes:
        - key: gcp.project
          value:
            stringValue: elastic-obs-integrations-dev
        - key: cloud.resource_id
          value:
            stringValue: networkmanagement.googleapis.com%2Fvpc_flows
        - key: gcp.resource_type
          value:
            stringValue: vpc_flow_logs_config
        - key: gcp.label.name
          value:
            stringValue: "joe-test-flow-logs-default"
        - key: gcp.label.location
          value:
            stringValue: global
        - key: gcp.label.resource_container
          value:
            stringValue: projects/774712120909
    scopeLogs:
      - logRecords:
          - attributes:
              - key: log.record.uid
                value:
                  stringValue: 16gybk0f1tkpe0
              - key: network.transport
                value:
                  stringValue: tcp
              - key: source.address
                value:
                  stringValue: 10.128.0.41
              - key: destination.address
                value:
                  stringValue: 35.191.240.41
              - key: source.port
                value:
                  intValue: "443"
              - key: destination.port
                value:
                  intValue: "40708"
              # good to here
              - key: gcp.vpc.flow.reporter
                value:
                  stringValue: SRC
              - key: gcp.vpc.flow.bytes_sent
                value:
                  intValue: "0"
              - key: gcp.vpc.flow.packets_sent
                value:
                  intValue: "640"
              - key: gcp.vpc.flow.start_time
                value:
                  stringValue: "2025-09-27T21:12:02.937646004Z"
              - key: gcp.vpc.flow.end_time
                value:
                  stringValue: "2025-09-27T21:12:02.937646004Z"
              - key: gcp.vpc.flow.source.instance.project.id
                value:
                  stringValue: elastic-obs-integrations-dev
              - key: gcp.vpc.flow.source.instance.vm.region
                value:
                  stringValue: us-central1
              - key: gcp.vpc.flow.source.instance.vm.name
                value:
                  stringValue: proxy-crest-new-group-b51k
              - key: gcp.vpc.flow.source.instance.vm.zone
                value:
                  stringValue: us-central1-a
              - key: gcp.vpc.flow.source.instance.managed_instance_group.name
                value:
                  stringValue: proxy-crest-new-group
              - key: gcp.vpc.flow.source.instance.managed_instance_group.zone
                value:
                  stringValue: us-central1-a
              - key: gcp.vpc.flow.source.project.id
                value:
                  stringValue: elastic-obs-integrations-dev
              - key: gcp.vpc.flow.source.subnet.name
                value:
                  stringValue: default
              - key: gcp.vpc.flow.source.subnet.region
                value:
                  stringValue: us-central1
              - key: gcp.vpc.flow.source.vpc.name
                value:
                  stringValue: default
              - key: gcp.vpc.flow.network_service.dscp
                value:
                  intValue: "32"
            body: {}
            observedTimeUnixNano: "1727500331539677207"
            timeUnixNano: "1727500331539677207"
        scope: {}

